modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/Swift3Codegen.java

```diff
@@ -582,7 +590,7 @@ private static CodegenModel reconcileProperties(CodegenModel codegenModel, Codeg
           Iterator<CodegenProperty> iterator = codegenProperties.iterator();
           while (iterator.hasNext()) {
               CodegenProperty codegenProperty = iterator.next();
-              if (codegenProperty.equals(parentModelCodegenProperty)) {
+              if (codegenProperty.baseName == parentModelCodegenProperty.baseName) {
```

Comment:
So moving forward, the `baseName` of the property becomes the key of the property, which is reasonable as I can't think of a language which allows both parent and child property/member having the same name.